### PR TITLE
acceptance tests setup

### PR DESCRIPTION
**Pull request by mchalski**
basic acceptance tests setup with a dummy passing test.

reminder: after merge check if a new image was pushed to dockerhub.

Issues: MEN-834

@mendersoftware/rndity @GregorioDiStefano

### CODE REVIEW

**Comment by mchalski** [comment]
@GregorioDiStefano althrough everything seems to be ok with the coverage reports, I have trouble interpreting the percentages: https://codecov.io/gh/mendersoftware/useradm/compare/d73f8025db0d439c9eaf9336550117c6fba09683...7eb0f87568aa7960b77a21ca90f1cc9ef0ec4a75

it says ~25% coverage for acceptance tests - but there's just one dummy test that never even calls the service. so what exactly *is* covered here?

then there's 66% patch coverage coverage - not sure where it comes from, this PR only sets the tests up...

maybe you can shed some light on it. or maybe @maciejmrowiec @kjaskiewiczz @bboozzoo ?

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+var runAcceptanceTests bool
+
+// used for parsing '-cli-args' for urfave/cli when running acceptance tests
+// this is because of a conflict between urfave/cli and regular go flags required for testing (can't mix the two)
+var cliArgsRaw string
+
+func init() {
+	// disable logging while running tests
+	log.Log.Out = ioutil.Discard
```
Comment (kjaskiewiczz) [inline main_test.go]: what about making this configurable by using this:
https://github.com/mendersoftware/go-lib-micro/blob/master/log/testing/log.go#L26

File: tests/run.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# tests are supposed to be located in the same directory as this file
+
+DIR=$(readlink -f $(dirname $0))
+
+export PYTHONDONTWRITEBYTECODE=1
+
+HOST=${HOST="mender-tenantadm:8080"}
```
Comment (kjaskiewiczz) [inline tests/run.sh]: mender-tenantadm ?

File: main_test.go (hunk 1 above)
Comment (mchalski) [inline main_test.go]: fixed

File: tests/run.sh (hunk 2 above)
Comment (mchalski) [inline tests/run.sh]: fixed

**Comment by kjaskiewiczz** [comment]
@GregorioDiStefano "~25% coverage for acceptance tests - but there's just one dummy test that never even calls the service"
but the service has been started and the start of the service touched ~25% of the code;
this is my understanding, at least;

**Pull request merged**